### PR TITLE
(DOCSP-17247): Realm SDKs Root Landing Page Update

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-17247): Realm SDKs Root Landing Page Update

### Staged Changes (Requires MongoDB Corp SSO)

- [MongoDB Realm SDKs](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/lp-main/sdk/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk.txt [hunk 1]
```diff
@@ -13,12 +13,30 @@ Realm SDKs
...
-The MongoDB Realm SDKs are client libraries that allow you to connect to
-local and synced Realm databases, authenticate users, and interact with
-serverless :ref:`MongoDB Realm <realm-cloud>` applications. The SDKs
-include {+client-database+}, an alternative to SQLite and Core Data, so
-you can get started with just the SDK -- no additional dependencies
-required.
+The MongoDB Realm SDKs are client libraries that allow you to connect to local
+and synced Realm databases, authenticate users, and interact with serverless
+:ref:`MongoDB Realm <realm-cloud>` applications. The SDKs include
+{+client-database+}, an alternative to SQLite, Core Data, and other embedded
```
Comment (MongoCaleb) [inline source/sdk.txt:19]: As with the other PR, I don't think we need to offer this as a replacement for SQLite.

File: source/sdk.txt [hunk 2]
```diff
@@ -13,12 +13,30 @@ Realm SDKs
...
+databases where your data model is your app's object classes instead of rows and
+columns in a table. You can get use Realm with just the SDK -- no additional
```
Comment (MongoCaleb) [inline source/sdk.txt:21]: And actually this sentence is really long and thus a bit confusing.

File: source/sdk.txt [hunk 3]
```diff
@@ -13,12 +13,30 @@ Realm SDKs
...
+dependencies required.
+
+Realm Database memory-maps and lazy-loads your data which makes querying fast
```
Comment (MongoCaleb) [inline source/sdk.txt:24]: I'm not 100% sure, but the hyphenated versions make me think they are nouns, not verbs. So maybe "Realm creates an in-memory map of your data and then lazily loads it, making..."

File: source/sdk.txt [hunk 4]
```diff
@@ -13,12 +13,30 @@ Realm SDKs
...
+and idiomatic for your language of choice and eliminates multithreading
+consistency bugs by making your objects thread-safe. 
+
+A database can either be local to the device or synced through MongoDB Realm.
+Both database types use the same APIs to define the schema, open the database,
```
Comment (MongoCaleb) [inline source/sdk.txt:29]: At least in .NET, the way you open a database depends on whether you are opening it with Sync or local-only.

Comment (nlarew) [reply source/sdk.txt:29]: And Realm Database doesn't exist at all in the Web SDK :) - I think for now we can gloss over small details like this but good catch!

Comment (MongoCaleb) [reply source/sdk.txt:28]: I guess. 🤷

**Review by MongoCaleb — APPROVED** [review]

**Pull request merged**